src/pages-conditional/community.js

```diff
@@ -0,0 +1,906 @@
+  twitter: `twitter`,
+  github: `github`,
+  youtube: `youtube`,
+}
+
+const forums = [
+  {
+    title: `e/ethereum`,
+    to: `https://www.reddit.com/r/ethereum/`,
+    description: `All things Ethereum`,
+    platform: socialPlatforms.reddit,
+  },
+  {
+    title: "r/ethfinance",
+    to: "https://www.reddit.com/r/ethfinance/",
+    description: "The financial side of Ethereum, including DeFi",
+    platform: socialPlatforms.reddit,
+  },
+  {
+    title: "r/ethdev",
+    to: "https://www.reddit.com/r/ethdev/",
+    description: "Focused on Ethereum development",
+    platform: socialPlatforms.reddit,
+  },
+  {
+    title: "r/ethtrader",
+    to: "https://www.reddit.com/r/ethtrader/",
+    description: "Trends & market analysis",
+    platform: socialPlatforms.reddit,
+  },
+  {
+    title: "r/ethstaker",
+    to: "https://www.reddit.com/r/ethstaker/",
+    description: "Welcome to all interested in staking on Ethereum",
+    platform: socialPlatforms.reddit,
+  },
+  {
+    title: "Ethereum Stackexchange",
+    to: "https://ethereum.stackexchange.com",
+    description: "Discussion and help for Ethereum developers",
+    platform: socialPlatforms.stackExchange,
+  },
+  {
+    title: "Ethereum Gitter",
+    to: "https://gitter.im/ethereum/home",
+    description: "Chat room for the Ethereum github repo",
+    platform: socialPlatforms.gitter,
+  },
+  {
+    title: "Ethereum Cat Herders",
+    to: "https://gitter.im/ethereum-cat-herders/community?source=orgpage",
+    description:
+      "Community oriented around offering project management support to Ethereum development",
+    platform: socialPlatforms.gitter,
+  },
+  {
+    title: "Ethereum Hackers",
+    to: "https://ethglobal.co/discord",
+    description:
+      "Discord chat run by ETHGlobal: an online community for Ethereum hackers all over the world",
+    platform: socialPlatforms.ethGlobal,
+  },
+  {
+    title: "CryptoDevs Discord",
+    to: "https://discord.gg/5W5tVb3",
+    description: "Ethereum development focused Discord community",
+    platform: socialPlatforms.discord,
+  },
+  {
+    title: "EthStaker Discord",
+    to: "https://discord.gg/3vbwcCYu35",
+    description: "Friendly place to chat and learn about staking on Ethereum",
+    platform: socialPlatforms.discord,
+  },
+  {
+    title: "Ethereum Foundation",
+    to: "https://www.youtube.com/channel/UCNOfzGXD_C9YMYmnefmPH0g",
+    description:
+      "Empowering developers to produce next generation decentralized applications in order to build a more free and trustworthy Internet.",
+    platform: socialPlatforms.youtube,
+  },
+  {
+    title: "Fellowship of Ethereum Magicians",
+    to: "https://ethereum-magicians.org",
+    description: "Community oriented around technical standards in Ethereum",
+  },
+  {
+    title: "Ethereum Research",
+    to: "https://ethresear.ch",
+    description:
+      "The most influential messageboard for cryptoeconomic research",
+  },
+]
+
+const events = [
```

Comment:
One idea to address [the concern](https://github.com/ethereum/ethereum-org-website/issues/1762#issuecomment-725652852) that moving from Markdown --> JS will make this harder to contribute - perhaps we move this & other source data into the src/data/ directory as a standalone CSV/YAML/JS file, so it's more apparent how to make edits/additions to this information. Might be a good approach to separate the data from the app logic, so e.g. we could have events, jobs, communities, meetups all sourced within /data/.

Thoughts?

Reply:
haha, just saw this after replying to your other note... I agree, 100% think these should go into their own files to make editing very straight forward